packages/fiori-elements-writer/src/index.ts

```diff
@@ -177,7 +176,7 @@ async function generate<T extends {}>(basePath: string, data: FioriElementsApp<T
     if (isEdmxProjectType) {
         // Add scripts to package.json only for non-CAP projects
         packageJson.scripts = Object.assign(packageJson.scripts ?? {}, {
-            ...getPackageJsonTasks({
+            ...generatePackageScripts({
```

Comment:
Technically this doesnt generate anything ( we refer to generators as something that writes something) so can we rename to `getPackageScripts` please

Reply:
changed name to `getPackageJsonTasks`